feat: override default dashboard via server-side WebSocket patching (#1276)
thomasloven/hass-browser_mod: documentation/configuration-panel.md

__IMPORTANT__: Like actions popups and notifications, this setting DOES NOT support templates.

### Default dashboard

> Set the default dashboard that is shown when you access Home Assistant.

Reviewer: ```suggestion
Set the default dashboard that is shown when you access Home Assistant base url (e.g. https://homeassistant.local/)
```

__IMPORTANT__: Like actions popups and notifications, this setting DOES NOT support templates.

### Default dashboard

> Set the default dashboard that is shown when you access `https://<your home assistant url>/` with nothing after the `/`.

Reviewer: ```suggestion
Set the default dashboard that is shown when you access Home Assistant.
```

Browser Mod supports three levels of default dashboard and applies them in the following priority order (highest first):

| Level | Scope | Overrides |
|-------|-------|-----------|

> | **Global** | All browsers, all users | Everything — browser-level, user-level, and native HA defaults |

Reviewer: ```suggestion
| **Global** | All browsers, all users | Everything — browser-level, user-level, and native Home Assistant defaults |
```

> | **Browser** | This registered browser, any user | User-level BM setting and native HA defaults |

Reviewer: ```suggestion
| **Browser** | This registered browser, any user | User-level Browser Mod setting and native Home Assistant defaults |
```

> | **User** | Any browser, this user | Native HA defaults only |

Reviewer: ```suggestion
| **User** | Any browser, this user | Native Home Assistant defaults only |
```

When a Global default dashboard is set it overrides all other settings, including any Browser- or User-level Browser Mod settings and any dashboard configured in the Home Assistant Dashboards settings or user profile. This override is enforced server-side, so it cannot be bypassed by changing local settings in the browser.

When a Browser- or User-level default dashboard is set (with no Global setting), Browser Mod takes control and the native Home Assistant dashboard picker in the user profile is disabled for those browsers or users. A notice will be shown in the HA user profile explaining that the default dashboard is managed by Browser Mod.

> When no Browser Mod default dashboard is set at any level, Home Assistant's own default dashboard settings (system dashboard and user profile dashboard) apply as normal.

Reviewer: All confusing. Consider if we need these at all and if such maybe one short sentance.
